src/frontend/src/views/Upload/components/Metadata/components/StaticTable/index.tsx

```diff
@@ -50,78 +62,132 @@ const validationSchema = object({
 export default function StaticTable({
   metadata,
   setIsValid,
+  hasImportedMetadataFile,
 }: Props): JSX.Element {
+  const [validationErrors, setValidationErrors] =
+    useState<Record<string, ValidationErrorRecord | null>>(EMPTY_OBJECT);
```

Comment:
it seems like the type is used in multiple places. Does it make sense to define this as a named type?